src/server.ts

```diff
@@ -8,6 +8,7 @@ import { appEnv } from './config'
 import { registerExpressApp } from './app'
 import errorHandler from 'errorhandler'
 import { env, EnvVariable } from './processEnv'
+import { NodemailerWrapper } from './mailer/nodemailerWrapper'
```

Comment:
file can be called `index.ts` so the import is always shorter:
```suggestion
import { mail } from './mailer'
```

Reply:
Complete